cluster.ml

```diff
@@ -30,6 +30,11 @@ let name s = sprintf "%s-%s" prefix s
 
 let gcloud_zone = env_exn "GCLOUD_ZONE"
 
+let additional_packages =
+  (try env_exn "ADDITIONAL_CLUSTER_PACKAGES"
+  with _ -> "")
+  |> String.split ~on:(`Character ',')
```

Comment:
Still, splitting on the empty string, returns a list with one element: `[""]` and `""` is not a package :)